src/app/public/modules/file-attachment/file-attachment.service.ts

```diff
@@ -48,20 +48,18 @@ export class SkyFileAttachmentService {
     return fileResults;
   }
 
-  public verifyDropFiles(files: any): boolean {
-    if (files.length > 1) {
-      return false;
-    }
-
+  /**
+   * Returns `true` if a directory is found in the provided `files` parameter.
+   */
+  public hasFolder(files: any): boolean {
```

Comment:
☝️ Two things happening here:
1. Renamed the method to `hasFolder` to be more transparent about its purpose.
2. Removed the `files.length > 1` check because you might want multiple files depending on what context this is called in. Removing that line also made this method have one single responsibility.